app/operations/mailers/user_added_to_project.rb

```diff
@@ -0,0 +1,10 @@
+module Mailers
+  class UserAddedToProject < Operation
+    integer :resource_id
+    array :roles
+
+    def execute
+      UserAddedToProjectMailerWorker.perform_async(api_user.id, resource_id, roles)
```

Comment:
With how little the operations themselves ended up doing, it might be fine if this line was just directly inside the controller. (and same with other operations). Saves one layer from the Controller -> Operation -> Worker -> Mailer pipeline that just seems to be passing along the method arguments directly.